theme/snippets/image-caption-module.liquid

```diff
@@ -7,7 +12,7 @@
   {% assign color = 'color-black' %}
 {% endif %}
 
-{% if image_module_image %}
+{% if image_module_first_image and variant != 'two-up' %}
   <div class="ImageCaptionModule flex flex-col 
     {% if variant == 'centered' %}
```

Comment:
**centered:** centered image (col-6), caption below
**left:** image on left, caption next to it
**full width:** full width, caption below
**two-up:** [ ] [ ] caption (img, img, caption)

Reply:
- Lines 15 - 43, are for the "single image" modules (one image with caption). Checking if the variant is not `'two-up' `because all the other variants are "single image" modules.
- Lines 45 - 61 are for the "two image" module (two images with caption).